packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue

```diff
@@ -363,21 +365,38 @@ function onCheckBoxChange(equation) {
 async function onRun() {
 	isOutputOpen.value = true;
 	isModelLoading.value = true;
-	const equations = clonedState.value.equations
+	const equationsText = clonedState.value.equations
 		.filter((e) => e.includeInProcess && !e.asset.extractionError)
 		.map((e) => e.asset.text);
+	const cleanedEquations = await getCleanedEquations(equationsText);
+	if (!cleanedEquations) return;
```

Comment:
You created for yourself two exception cases:
- null
- empty array

This seem like it will handle null but not the empty-array case, are we okay with sending an empty-array down the road or do we need to stop?

Reply:
It depends if an empty array is a valid response.
If there is an error within `getCleanedEquations()` it should return `null`.

Reply:
I always disliked how `null` is being leveraged to indicate errors in the front-end, it makes more sense if the callers handle exceptions as they see fit, so they can choose to halt or try to continue. But this is a different discussion.

Empty array is a valid response for "get-clean-equations" API call, but my guess is that it is probably not a valid input for "equation-to-model". since it doesn't make sense to create a model out of `[]`.